templates/index.html

```diff
@@ -1,28 +1,184 @@
+        </div>
+        <div id="navbar" class="collapse navbar-collapse">
+          <ul class="nav navbar-nav">
+            <!-- <li class="active"><a href="#">Home</a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#contact">Contact</a></li> -->
+          </ul>
+        </div><!--/.nav-collapse -->
+      </div>
+    </nav>
+    <div class="container">
+      <h1>Cronitor Job Dashboard</h1>
+      <input type="text" id="myInput" onkeyup="myFunction()" placeholder="Search for Servers ..." title="Type in a name">
+      <table class="table" id="myTable">
+        <thead>
+            <tr class=header>
+                <th onclick="sort_table(people, 0, asc1); asc1 *= -1; asc2 = 1; asc3 = 1; asc4 = 1">Name</th>
+                <th onclick="sort_table(people, 1, asc2); asc2 *= -1; asc3 = 1; asc4 = 1; asc1 = 1">Last Run</th>
+                <th onclick="sort_table(people, 2, asc3); asc3 *= -1; asc4 = 1; asc1 = 1; asc2 = 1">Status</th>
+                <th onclick="sort_table(people, 2, asc4); asc4 *= -1; asc1 = 1; asc2 = 1; asc3 = 1">Log</th>
+            </tr>
+        </thead>
+        <tbody id="people">
+        {% for job in jobs.jobs %}
+        {% set ent = job.latest_entry %}
+          <tr>
+            <td>{{ job.name }}</td>
+            <td>{{ ent.timestamp }}</td>
+            <td class="job_{{job.status}}">{{ job.status }}</td>
+            <td>
+              <!-- <a href="/log/{{ url_escape(job.name) }}/{{ url_escape(ent.timestamp.isoformat()) }}">newest</a> -->
+              <a href="/log/{{ url_escape(job.name) }}">LOGFILES</a>
+          </td>
+        {% end %}
+        </tbody>
+      </table>
+    </div>
+
+    <!-- Bootstrap core JavaScript
+    ================================================== -->
+    <!-- Placed at the end of the document so the pages load faster -->
+    <!-- <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <script>window.jQuery || document.write('<script src="../../assets/js/vendor/jquery.min.js"><\/script>')</script>
+    <script src="../../dist/js/bootstrap.min.js"></script>
+    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
+    <!-- <script src="../../assets/js/ie10-viewport-bug-workaround.js"></script> -->
+    <script>
+      function myFunction() {
+        var input, filter, table, tr, td, i;
+        input = document.getElementById("myInput");
+        filter = input.value.toUpperCase();
+        table = document.getElementById("myTable");
+        tr = table.getElementsByTagName("tr");
+        for (i = 0; i < tr.length; i++) {
+          td = tr[i].getElementsByTagName("td")[0];
+          if (td) {
+            if (td.innerHTML.toUpperCase().indexOf(filter) > -1) {
+              tr[i].style.display = "";
+            } else {
+              tr[i].style.display = "none";
+            }
+          }
+        }
+      }
+      </script>
```

Comment:
A lot of this is common to all the templates, so it could be factored out into a base template, and included here using template inheritance.  (I hadn't done it myself previously, since there wasn't much in common.)

If you haven't used Tornado templates before, you can see how inheritance works here:

http://www.tornadoweb.org/en/stable/template.html

Look for the `{% extends ... %}` and `{% block ... %}` commands.